src/Nethermind/Nethermind.Cli/Program.cs

```diff
@@ -30,45 +33,67 @@ namespace Nethermind.Cli
 {
     public static class Program
     {
-        private static readonly ColorScheme ColorScheme = DraculaColorScheme.Instance;
-        private static readonly CliConsole CliConsole = new CliConsole();
-        private static readonly StatementHistoryManager HistoryManager = new StatementHistoryManager(CliConsole);
-
-        private static readonly ILogManager LogManager = new OneLoggerLogManager(new CliLogger(CliConsole));
+        private static readonly Dictionary<string, ColorScheme> _availableColorSchemes = new(){  {"basic", BasicColorScheme.Instance },
+                                                                                                {"dracula", DraculaColorScheme.Instance }};
         private static readonly IJsonSerializer Serializer = new EthereumJsonSerializer();
 
-        private static readonly Terminal Terminal = CliConsole.Init(DraculaColorScheme.Instance);
-        private static readonly ICliEngine Engine = new CliEngine(CliConsole);
-        private static readonly INodeManager NodeManager = new NodeManager(Engine, Serializer, CliConsole, LogManager);
+        public static void Main(string[] args)
+        {
+            CommandLineApplication app = new() { Name = "Nethermind.Cli" };
+            _ = app.HelpOption("-?|-h|--help");
 
-        private static readonly CliModuleLoader ModuleLoader = new CliModuleLoader(Engine, NodeManager, CliConsole);
+            var colorSchemeOption = app.Option("-cs|--colorScheme <colorScheme>", "Color Scheme. Possible values: Basic|Dracula", CommandOptionType.SingleValue);
 
-        public static void Main()
-        {
-            RegisterConverters();
-            Engine.JintEngine.SetValue("serialize", new Action<JsValue>(v =>
+            app.OnExecute(() =>
+            {
+                ColorScheme? cs;
+                ICliConsole cliConsole = colorSchemeOption.HasValue() && (cs = MapColorScheme(colorSchemeOption.Value())) != null
+                    ? new ColorfulCliConsole(cs)
+                    : new CliConsole();
+
+                var historyManager = new StatementHistoryManager(cliConsole);
```

Comment:
NP: Inconsistent use of `var` or explicit type. I personally prefer `var`, but intellij seems to like explicit type better. In any case, please be consistent within the same file.